Documentation/compatibility/wpf-MarkupCompiler-default-hash-algorithm-is-now-SHA256.md

```diff
@@ -0,0 +1,41 @@
+## The default hash algorithm for WPF's Markup Compiler is now SHA256
+
+### Scope
+Transparent
+
+### Version Introduced
+4.7.2
+
+### Source Analyzer Status
+NotPlanned
+
+### Change Description
+The WPF MarkupCompiler provides compilation services for XAML markup files.  In the .NET Framework 4.7.1 and earlier versions, the default hash algorithm used for checksums was SHA1.
+Due to recent security concerns with SHA1, this default has been changed to SHA256 starting with the .NET Framework 4.7.2.  This change affects all checksum generation for markup files during compilation.
+
+- [X] Quirked
+- [ ] Build-time break
+
+### Recommended Action
```

Comment:
It's best to break these out into two separate paragraphs and two separate code blocks, starting with the developer who wants to revert to 4.7,1 behavior  For the second block, you should mention that the app has to run on .NET Framework 4.7.2 or later while targeting a previous version.